Documentation/getting-started.md

```diff
@@ -1,6 +1,6 @@
 ## Getting started
 
-The Adobe Experience Platform Media for Edge Network mobile extension has the following dependencies, which must be installed prior to installing the extension:
+The Adobe Streaming Media for Edge Network mobile extension has the following dependencies, which must be installed prior to installing the extension:
 - [MobileCore](https://github.com/adobe/aepsdk-core-android)
 - [Edge](https://github.com/adobe/aepsdk-edge-android)
 - [EdgeIdentity](https://github.com/adobe/aepsdk-edgeidentity-android)
```

Comment:
for the configuration setting below can we link to the developer docs instead similar to the other extensions?

Reply:
I thought we decided to link Github links. I see Github links in edgeIdentity as well
https://raw.githubusercontent.com/adobe/aepsdk-edgeidentity-android/main/Documentation/getting-started.md

Reply:
I meant for the Configuration section bellow L8-L44, but I could not comment on that line

Reply:
You mean the entire section to replace with the link.

Reply:
I mean to have something similar to the Identity for Edge Network extension where you configure it in the Data Collection UI and have a link for external steps. As we discussed we can take this one in a separate PR once the docs link is available